Program.cs

```diff
@@ -1,7 +1,33 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿internal class Program {
+   private static void Main (string[] args) {
+      int a = 1;

```

Comment:
You could combine multiple variable declarations of same data type like "int a = 1, z = 101;"